utility/src/main/java/org/oppia/android/util/profile/NameValidatorUtil.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.android.util.profile
+
+/** Utility class for [NameValidator]. */
+object NameValidatorUtil {
```

Comment:
I'm not sure I see the point of this class--why not include these properties within NameValidator itself?